maxim/logger/langchain/utils.py

```
@@ -610,6 +612,13 @@ def parse_langchain_messages(
                                 "tool_call_id": message.tool_call_id,
                             }
                         )
+                    elif message_type.endswith("ChatMessage"):
+                        messages.append(
+                            {
+                                "role": message.role,
+                                "content": message.content or "",
+                            }
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)